### PR TITLE
chore(ci): Add workflows to enable continuous delivery w/ GHA

### PR DESCRIPTION
**Pull request by joshuef**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,42 @@
...
+* **bump:** test GHA ([734bc08](https://github.com/joshuef/safe-nd/commit/734bc0841b5d5f0ce1da4f8633d30f8410ce9579))
+
+### 0.8.1 (2020-07-20)
+
+
+### Features
+
+* **audit:** add scheduled security audit scan ([4725d13](https://github.com/joshuef/safe-nd/commit/4725d13ee7a473f46bf2e119d5021b6ab5f8fb71))
+* **bump:** test GHA ([8142f8a](https://github.com/joshuef/safe-nd/commit/8142f8ad137a38f43ba43d33bb46e6f1f320b950))
+* **bump:** test GHA ([734bc08](https://github.com/joshuef/safe-nd/commit/734bc0841b5d5f0ce1da4f8633d30f8410ce9579))
```
Comment (S-Coyle) [inline CHANGELOG.md]: The 0.8.1 release section is repeated 4 times

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,42 @@
...
 
 ## [0.10.1]
```
Comment (S-Coyle) [inline CHANGELOG.md:5]: 0.10.1, 0.10.0 and 0.9.0 should be above 0.8.1. Is this because 0.8.1 was missing from the changelog?

Comment (joshuef) [reply CHANGELOG.md:5]: ah, all that needs cleaned out ups. 

That's me having run the action a bunch of times w/ different versions and forgetting to clean the changelog :|

Comment (joshuef) [reply CHANGELOG.md:5]: (and my repo not having latest tags, so opting for 8.1)

File: .github/workflows/bump_version.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  bump:
+    runs-on: ubuntu-latest
+    if: "!startsWith(github.event.head_commit.message, 'chore(release):')"
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: '0'
+ 
+      - name: Bump Version
+        uses: joshuef/auto-rust-version-bumper@v0.10
```
Comment (S-Coyle) [inline .github/workflows/bump_version.yml]: Should now be v1

Comment (S-Coyle) [reply .github/workflows/bump_version.yml]: and should now be `maidsafe/` 😃

**Review by S-Coyle — APPROVED** [review]
👍

**Pull request merged**